bawan-shna-abdulrahman/src/Components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Navbar, Nav } from "react-bootstrap";
+import Search from "./Search";
+
+export default function Header(props) {
+  return (
+    <div>
+      <Navbar bg="dark" expand="lg">
+        <Navbar.Brand href="#home" style={{ color: "white" }}>
+          React App
+        </Navbar.Brand>
+        <Navbar.Toggle aria-controls="basic-Navbar-nav" />
+        <Navbar.Collapse id="basic-Navbar-nav">
+          <Nav className="mr-auto">
+            <Nav.Link href="#home" style={{ color: "white" }}>
+              Home
+            </Nav.Link>
+          </Nav>
+          <Search
+            handleQuery={props.handleQuery}
+            handleMovies={props.handleMovies}
+            isLoading={props.isLoading}
+            setIsLoading={props.setIsLoading}
+          />
```

Comment:
you can use the destructure here
```suggestion
export default function Header(props) {

const {handleQuery, handleMovies, isLoading, setIsLoading } = props

  return (
    <div>
      <Navbar bg="dark" expand="lg">
        <Navbar.Brand href="#home" style={{ color: "white" }}>
          React App
        </Navbar.Brand>
        <Navbar.Toggle aria-controls="basic-Navbar-nav" />
        <Navbar.Collapse id="basic-Navbar-nav">
          <Nav className="mr-auto">
            <Nav.Link href="#home" style={{ color: "white" }}>
              Home
            </Nav.Link>
          </Nav>
          <Search
            handleQuery={handleQuery}
            handleMovies={handleMovies}
            isLoading={isLoading}
            setIsLoading={setIsLoading}
          />
```